Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/export.js

```diff
@@ -5,10 +5,58 @@ if (AppDependencies !== undefined) {
     AppDependencies.push(moduleName);
 }
 
-angular.module(moduleName, ['ngSanitize'])
+angular.module(moduleName, ['ui.grid.cellNav', 'ui.grid.edit', 'ui.grid.validate'])
+    .config(['$stateProvider', function ($stateProvider) {
+        $stateProvider.state('workspace.exportModule', {
+            url: '/exportNew',
+            templateUrl: '$(Platform)/Scripts/common/templates/home.tpl.html',
+            controller: ['$scope', 'platformWebApp.bladeNavigationService', function ($scope, bladeNavigationService) {
+                var blade = {
+                    id: 'exportNew',
+                    controller: 'virtoCommerce.exportModule.exportGroupsController',
+                    template: 'Modules/$(VirtoCommerce.Export)/Scripts/blades/exportGroups.tpl.html',
+                    isClosingDisabled: true
+                };
+                bladeNavigationService.showBlade(blade);
+            }]
+        });
+    }])
     .run(
-        [
-            'platformWebApp.toolbarService', 'platformWebApp.bladeNavigationService', 'platformWebApp.widgetService',
-            function(toolbarService, bladeNavigationService, widgetService) {
-            }
-        ]);
+        ['$http', '$compile', 'platformWebApp.mainMenuService', 'platformWebApp.widgetService', '$state', 'platformWebApp.authService'
+            , function ($http, $compile, mainMenuService, widgetService, $state, authService) {
+                //Register module in main menu
+                var menuItem = {
+                    path: 'configuration/export',
+                    icon: 'fa fa-database',
+                    title: 'export.main-menu-title',
+                    priority: 30,
+                    action: function () { $state.go('workspace.exportModule'); },
+                    permission: 'export:access'
+                };
+                mainMenuService.addMenuItem(menuItem);
+
+                ////Register item prices widget
```

Comment:
Please remove. This code is module specific, and will be included in modules which will implement export.